testnets/qubeticstestnet/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "qubeticstestnet",
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Qubetics Chain",
+      "denom_units": [
+        {
+          "denom": "tics",
+          "exponent": 0
+        },
+        {
+          "denom": "TICS",
+          "exponent": 18
+        }
+      ],
+      "base": "tics",
+      "name": "Qubetics Testnet",
+      "display": "tics",
```

Comment:
Please update this also to 'TICS' so interfaces know to display the asset with 18 exponent